openshift_images/postgresql.adoc

```diff
@@ -10,89 +10,154 @@
 
 == Usage
-To create a container running PostgreSQL, you need to run following commands.
+To create a container running PostgreSQL, you must run the following commands.
+
+*RHEL-7 image*
 
-=== RHEL-7 image
+****
+`$ docker run -d --name _<database>_ -e POSTGRESQL_USER=_<user>_ -e POSTGRESQL_PASSWORD=_<password>_ -e POSTGRESQL_DATABASE=_<database>_ -p 5432:5432 openshift/postgresql-92-rhel7`
+****
 
-----
-$ docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 openshift/postgresql-92-rhel7
-----
+*CentOS-7 image*
 
-=== CentOS-7 image
+****
+`$ docker run -d --name _<database>_ -e POSTGRESQL_USER=_<user>_ -e POSTGRESQL_PASSWORD=_<password>_ -e POSTGRESQL_DATABASE=_<database>_ -p 5432:5432 openshift/postgresql-92-centos7`
+****
 
-----
-$ docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 openshift/postgresql-92-centos7
-----
+== Testing
+To access and test if the application is running:
 
-This will create a container named `postgresql_database`, with credentials `user:pass` and database `db`. To access and test if the application is running, run:
-----
-$ psql postgresql://user:pass@127.0.0.1:5432/db <<< "SELECT 1;"
-----
+****
+`$ psql postgresql://_<user>_:<password>@127.0.0.1:5432/_<database>_ <<< "SELECT 1;"`
+****
 
+== Stopping the Container
 To properly stop the container:
-----
-$ docker stop postgresql_database
-----
+
+****
+`$ docker stop _<database>_`
+****
 
 == Environment variables
 
-* *POSTGRESQL_USER* - This variable specifies username for database user that will be created for your application to use.
+The image recognizes the following environment variables that you can set
+during initialization with this command:
 
-* *POSTGRESQL_PASSWORD* - Password for the *POSTGRESQL_USER*.
+****
+`$ docker run -e VAR=_<variable>_`
+****
 
-* *POSTGRESQL_DATABASE* - Name of the database that will be created. *POSTGRESQL_USER* will have full rights to the database.
+.Repository Organization
+[cols="4a,6a",options="header"]
+|===
 
-* *POSTGRESQL_ADMIN_PASSWORD* - Password for the postgres user. This variable is optional. If not set, remote login to the postgres account will not be possible. Local connections from inside the container are always allowed without a password.
+|Variable name |Description
 
-== Volumes
+|`*POSTGRESQL_USER*`
+|Username for the PostgreSQL account to be created. This user has full rights to the database.
 
-* */var/lib/pgsql/data* - This is the database cluster directory where PostgreSQL stores database files.
+|`*POSTGRESQL_PASSWORD*`
+|Password for the user account.
+
+|`*POSTGRESQL_DATABASE*`
+|Database name.
+
+|`*POSTGRESQL_ADMIN_PASSWORD*`
+|Optional password for the Postgres admin user. If this is not set, then remote login to the Postgres account is not possible. Local connections from within the container are always permitted without a password.
```

Comment:
Postgres without capital first letter. It is a user name, so it might be in the `` quotes possibly?